src/components/PasswordStrengthMeter/index.tsx

```diff
@@ -62,45 +57,49 @@ export type PasswordStrengthMeterProps = {
   userInputs?: string[]
 }
 
-const PasswordStrengthMeter: FunctionComponent<PasswordStrengthMeterProps> = ({
+const PasswordStrengthMeter = ({
   password = '',
   onChange,
   strength,
   title,
   estimate = () => ({ score: 0 }),
   userInputs = [],
-  ...props
-}) => {
+}: PasswordStrengthMeterProps): JSX.Element => {
   const [score, setScore] = useState<number>(0)
   const [backgroundColor, setBackgroundColor] = useState<string>(
-    strength[0].color,
+    strength?.[0]?.color || 'green',
```

Comment:
Weird, strength should never be undefined no ?

Reply:
Yeah my bad it wasn't defined in default story (causing storybook to crash) and I changed it there but forgot to remove `?` here